SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -136,3 +136,29 @@ void CDriver::BoundaryConditionsUpdate() {
     geometry_container[iZone][INST_0][MESH_0]->UpdateCustomBoundaryConditions(geometry_container[iZone][INST_0],config_container[iZone]);
   }
 }
+
+////////////////////////////////////////////////////////////////////////////////
+/* Functions related to dynamic mesh */
+////////////////////////////////////////////////////////////////////////////////
+
+void CDriver::SetTranslationRate(passivedouble xDot, passivedouble yDot, passivedouble zDot) {
+  if (rank == MASTER_NODE) {
+    cout << endl << " Setting new translational velocity in x, y, z direction.";
+  }
+  for (iZone = 0; iZone < nZone; iZone++) {
+    config_container[iZone]->SetTranslation_Rate(0, xDot);
+    config_container[iZone]->SetTranslation_Rate(1, yDot);
+    config_container[iZone]->SetTranslation_Rate(2, zDot);
+  }
+}
+
+void CDriver::SetRotationRate(passivedouble rot_x, passivedouble rot_y, passivedouble rot_z) {
+  if (rank == MASTER_NODE) {
+    cout << endl << " Setting new angular velocity about x, y, z axes.";
+  }
```

Comment:
We don't print messages for other things we set via the python interface.
```suggestion
```

Reply:
Removed in 4f00bcc